### DISCUSSION TITLE
Help: Forecasters like VAR in statsmodels

### DISCUSSION THREAD

**Discussion by WenjieZ** [alan-turing-institute/sktime#4385] [Q&A]
Are there any forecasters in this package like VAR, which support multi-dimensional time series with inter-dimensional correlation?

**Comment by fkiraly** [comment]
The command to get these is:

```python
from sktime.registry import all_estimators

all_estimators("forecaster", filter_tags={"scitype:y": ["both", "multivariate"]}, as_dataframe=True)
```

If you discount composites where the tag indicates potential capability depending on components, the current forecasters in `sktime` (0.16.1) that are genuinely multivariate are (according to tags):

`DynamicFactor`, `VAR`, `VARMAX`, `VECM`.

I'll also move this to GitHub discussions as it is a Q&A issue.
https://github.com/sktime/sktime/discussions

(feel free to follow up there, and mark as resolved if this has answered your question!)

**Comment by fkiraly** [comment]
On another note, you can also create pipelines that have make use of multivariate correlation by virtue of transformers applied.

For instance, you could pipeline `ARIMA` with a `Reconciler` or `PCA`, that would make it genuinely multivariate by the transformer.
